src/Libraries/RevitNodes/Elements/Material.cs

```diff
@@ -101,6 +101,186 @@ public static Material ByName(string name)
 
         #endregion
 
+        #region Properties
+
+        /// <summary>
+        /// Get Material Name
+        /// </summary>
+        /// <returns></returns>
+        public string Name
+        {
+            get { return this.InternalMaterial.Name; }
+        }
+
+        /// <summary>
+        /// Get Shininess
+        /// </summary>
+        public int Shininess
+        {
+            get { return this.InternalMaterial.Shininess; }
+        }
+
+        /// <summary>
+        /// Get Smoothness
+        /// </summary>
+        public int Smoothness
+        {
+            get { return this.InternalMaterial.Smoothness; }
+        }
+
+        /// <summary>
+        /// Get Transparency
+        /// </summary>
+        public int Transparency
+        {
+            get { return this.InternalMaterial.Transparency; }
+        }
+
+        /// <summary>
+        /// Get SurfacePatternColor
+        /// </summary>
+        public DSCore.Color SurfacePatternColor
```

Comment:
did you have to add a new reference for this or we already referenced - just curious.